### DISCUSSION TITLE
Changing settings in editor beatmap setup doesn't prompt for a save when exiting

### DISCUSSION THREAD

**Discussion by Walavouchey** [ppy/osu#18454] [Q&A] [answered]
Changing metadata, difficulty settings, combo colour, etc. doesn't cause a save prompt when exiting the editor (through either `Esc` or `File` -> `Exit`).

I haven't shown this in the video, but if you do this with the background or audio track, it'll clear them instead (the latter causing a "Waveform failed to load" error).

https://user-images.githubusercontent.com/36758269/170816353-e4940de9-ffc2-4905-a0b5-f278a98dd8fa.mp4

[logs.zip](https://github.com/ppy/osu/files/8790736/logs.zip)

version: latest release (2022.528.0)

**Comment by bdach** [comment, marked as answer]
already known and tracked at #10314

**Reply by Walavouchey** [reply]
ah ty, couldn't find it
